tests/unit/test_expression_tree/test_symbol.py

```diff
@@ -339,10 +339,10 @@ def test_has_spatial_derivatives(self):
 
     def test_orphans(self):
         a = pybamm.Scalar(1)
-        b = pybamm.Scalar(2)
-        sum = a + b
+        b = pybamm.Parameter("b")
+        summ = a + b
```

Comment:
?

Reply:
again simplify on creation broke the test